hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleCreateHandler.java

```diff
@@ -41,21 +41,15 @@ public class ScheduleCreateHandler extends AbstractScheduleHandler implements Tr
     @Inject
     public ScheduleCreateHandler() {
         // Exists for injection
+        super(null);
```

Comment:
Likely this is incorrect.  I suspect the other (newly added) constructor needs to be annotated for injection (perhaps best done once Hendrik's Dagger module improvements are merged).
Maybe add a comment like the following:
`// @todo('6249') This constructor should be removed and the @Inject annotation added to the remaining constructor`

Reply:
Yes, this is definitely incorrect and needs to be changed once DI was fixed. Added the comment.